File: app/src/main/java/net/sourcefusion/agiletools/sprintcalendar/DefaultDateProvider.java
package net.sourcefusion.agiletools.sprintcalendar;

import org.joda.time.LocalDate;

class DefaultDateProvider implements DateProvider {

    @Override
    public boolean isToday(LocalDate date) {
        return getToday().compareTo(date) == 0;
    }

    @Override
    public LocalDate getToday() {
        return new LocalDate();
    }
}

File: app/src/main/kotlin/net/sourcefusion/agiletools/sprintcalendar/DefaultDateProvider.kt
package net.sourcefusion.agiletools.sprintcalendar

import org.joda.time.LocalDate
import org.joda.time.LocalTime

internal class DefaultDateProvider : DateProvider {
    override fun isToday(date: LocalDate): Boolean {
        return today.compareTo(date) == 0
    }

    override val today: LocalDate
        get() = LocalDate()

    override val time: LocalTime
        get() = LocalTime()
}
